<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>win1</class>
 <widget class="QMainWindow" name="win1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>163</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>261</width>
    <height>163</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>261</width>
    <height>163</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Select columns</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(29, 29, 29);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_date">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>11</y>
      <width>40</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(79, 79, 79);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Date:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_date">
    <property name="geometry">
     <rect>
      <x>78</x>
      <y>14</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(79, 79, 79);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_activity">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>37</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(79, 79, 79);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Activity:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_activity">
    <property name="geometry">
     <rect>
      <x>78</x>
      <y>39</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(79, 79, 79);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_details">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>63</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(79, 79, 79);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Details:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_details">
    <property name="geometry">
     <rect>
      <x>77</x>
      <y>65</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(79, 79, 79);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_time">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>89</y>
      <width>44</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(79, 79, 79);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Time:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_time">
    <property name="geometry">
     <rect>
      <x>77</x>
      <y>91</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(79, 79, 79);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_accept">
    <property name="geometry">
     <rect>
      <x>73</x>
      <y>121</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(79, 79, 79);</string>
    </property>
    <property name="text">
     <string>Accept</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cancel">
    <property name="geometry">
     <rect>
      <x>164</x>
      <y>121</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(79, 79, 79);</string>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>comboBox_details</tabstop>
  <tabstop>comboBox_date</tabstop>
  <tabstop>comboBox_activity</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
